src/EmbeddedVsix.UnitTests/TypeScriptRuleDefinitions_SanityChecks.cs

```diff
@@ -95,6 +95,6 @@ private static void CheckRules(string language, IEnumerable<RuleDefinition> rule
             ruleDefinition.RuleKey.EndsWith(":S2260");
 
         private static bool HasDefaultParameters(RuleDefinition ruleDefinition) =>
-                ruleDefinition.DefaultParams.Length > 0;
+                ruleDefinition.DefaultParams?.Length > 0;
```

Comment:
Not sure if this is the best way to solve the error in the test.

Reply:
If I understand correctly, you've changed the test so that it won't fail if a rule doesn't have default parameters.

Does our product handle this case? Presumably before it was assuming that the default parameters might be empty but not null.

Is this actually a change of format by the JS/TS bubble, and if so was it intended?